### PR TITLE
Make sure VS doesn't messup with Splash-screen generation

### PR DESCRIPTION
**Pull request by pedrojesus-work**
Fixes: #243

## PR Type

- Bugfix
What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by dansiegel — APPROVED** [review]

File: src/.nuspec/Uno.Resizetizer.targets [hunk 1]
```diff
@@ -471,7 +471,10 @@
 			Outputs="$(_UnoResizetizerStampFile)"
 			AfterTargets="$(UnoResizetizeAfterTargets)"
 			BeforeTargets="$(UnoResizetizeBeforeTargets)"
-			DependsOnTargets="$(UnoResizetizeDependsOnTargets)">
+			DependsOnTargets="$(UnoResizetizeDependsOnTargets)"
+			Condition="
```
Comment (jeromelaban) [inline src/.nuspec/Uno.Resizetizer.targets:475]: @dansiegel @pedrojesus-work the comment does not match anymore

Comment (jeromelaban) [reply src/.nuspec/Uno.Resizetizer.targets:475]: Scratch that, it's android only.

Comment (pictos) [reply src/.nuspec/Uno.Resizetizer.targets:475]: So are we good?

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/1.4

**Pull request merged**